<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceOptionsWidget</class>
 <widget class="QWidget" name="ServiceOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Service</string>
  </property>
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item>
          <widget class="QPlainTextEdit" name="plainServiceDescription">
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="plainText">
            <string>HostMonitor uses these settings only when started as a Win32 interactive service.
To install HostMonitor as service use command line parameter &quot;/InstallService&quot; (e.g. &quot;hostmon.exe /InstallService&quot;). </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="boxServiceAccount">
           <property name="title">
            <string> Log on as  </string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_16">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QRadioButton" name="rbServiceLocalAccount">
              <property name="text">
               <string>Local system acount</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QRadioButton" name="rbServiceUseAccount">
                <property name="text">
                 <string>Use account</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editServiceUser"/>
              </item>
              <item>
               <widget class="QLabel" name="lblServicePassword">
                <property name="text">
                 <string>password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="editServicePassword"/>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frmStartupDelay">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="lblServiceStartupDelay">
              <property name="text">
               <string>Startup delay</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinServiceStartupDelay"/>
            </item>
            <item>
             <widget class="QLabel" name="lblServiceMin">
              <property name="text">
               <string>min</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="boxServiceSaveMode">
           <property name="title">
            <string> When service is being stopped  </string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="topMargin">
             <number>6</number>
            </property>
            <property name="bottomMargin">
             <number>6</number>
            </property>
            <item>
             <widget class="QRadioButton" name="rbServiceSaveInHML">
              <property name="text">
               <string>save changes in HML file</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="rbServiceDiscardChanges">
              <property name="text">
               <string>discard all changes</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>

 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
